prowler/lib/mutelist/mutelist.py

```diff
@@ -317,12 +321,19 @@ def is_item_matched(matched_items, finding_items):
         try:
             is_item_matched = False
             if matched_items and (finding_items or finding_items == ""):
+                if tag:
+                    is_item_matched = True
```

Comment:
What is the point of this?

Reply:
To apply the AND logic, I'll explain it in the comment below.